README.md

```diff
@@ -155,7 +155,12 @@ If you don't have a `.prettierrc` config, please add it (for example):
 
 This code is licensed under the [MIT](/LICENSE) License.
 
-### Contributions
+### Contributors
```

Comment:
Please keep the section `### Contributions`

And add the section `### Contributors` at the bottom of readme